src/vm/ceemain.h

```diff
@@ -16,14 +16,15 @@
 #define CEEMain_H
 
 #include <windef.h> // for HFILE, HANDLE, HMODULE
+#include <corbundle.h>
 
 class EEDbgInterfaceImpl;
 
 // Ensure the EE is started up.
-HRESULT EnsureEEStarted(COINITIEE flags);
+HRESULT EnsureEEStarted(COINITIEE flags, const BundleInfo* bundleInfo);
```

Comment:
Do you really need to pass the BundleInfo* everywhere? Can we just store it in a static and get everybody who needs to fetch it from the static?

Reply:
:+1 at the very least we should pass it along at the same place where we pass runtime properties - which is `CreateAppDomainWithManager`, but ideally this would not go through the COM APIs at all. As mentioned in other comments, I think this should be stored only in one place - the AppDomain is probably as good a place as any right now.